.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on:
+  push:
+  pull_request:
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  test-cloud-gallery:
+    runs-on: ${{ fromJSON('["ubuntu-latest", "self-hosted"]')[github.repository == 'github/docs-internal'] }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.13.x
+
+      - name: Install
+        working-directory: ./cloud-gallery
+        run: npm ci
+
+
+      - name: Run tests
+        working-directory: ./cloud-gallery
+        run: npm run -w helpers build && npm run test
```

Comment:
not a big deal but I find `test` here pretty misleading since we don't actually have any proper testing

Reply:
I'll change it to check